src/Microsoft.Management.UI.Internal/ShowCommand/ViewModel/CommandViewModel.cs

```diff
@@ -24,7 +24,7 @@ namespace Microsoft.PowerShell.Commands.ShowCommandInternal
     /// Contains information about a cmdlet's Shard ParameterSet,
     /// ParameterSets, Parameters, Common Parameters and error message.
     /// </summary>
-    public class CommandViewModel : INotifyPropertyChanged
+    public sealed class CommandViewModel : INotifyPropertyChanged
```

Comment:
Another public API. @SteveL-MSFT can we change this?

Reply:
Namespace is `Microsoft.PowerShell.Commands.ShowCommandInternal` so change is Bucket 4: "Clearly Non-Public"?

Reply:
The code is frozen. Please revert the change.

Reply:
When was `Microsoft.Management.UI.Internal` frozen?

Reply:
The code was added only for backward compatibility. All new work will be in PowerShell/GraphicalTools repo.